src/Symfony/Component/HttpFoundation/File/File.php

```diff
@@ -133,4 +133,18 @@ protected function getName($name)
 
         return $originalName;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSize()
+    {
+        // Get the file size from a stream rather than from disk in case it is changed later by
+        // stream wrappers or stream filters.
+        $file = fopen($this->getPathname(), 'rb');
+        $fstat = fstat($file);
```

Comment:
To do that I think we'd need to write a test that is actually encrypting files so that the size is different on disc. I assume that's the best way to go about doing this?